packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -303,11 +303,17 @@ - (void)createCameraOnSessionQueueWithCreateMethodCall:(FlutterMethodCall *)crea
     if (!strongSelf) return;
 
     NSString *cameraName = createMethodCall.arguments[@"cameraName"];
+    NSNumber *fps = createMethodCall.arguments[@"fps"];
+    NSNumber *videoBitrate = createMethodCall.arguments[@"videoBitrate"];
+    NSNumber *audioBitrate = createMethodCall.arguments[@"audioBitrate"];
     NSString *resolutionPreset = createMethodCall.arguments[@"resolutionPreset"];
     NSNumber *enableAudio = createMethodCall.arguments[@"enableAudio"];
     NSError *error;
     FLTCam *cam = [[FLTCam alloc] initWithCameraName:cameraName
                                     resolutionPreset:resolutionPreset
+                                                 fps:fps
+                                        videoBitrate:videoBitrate
+                                        audioBitrate:audioBitrate
```

Comment:
can we unwrap these NSNumbers from the caller? Or do we need the nullability inside FLTCam?

Reply:
Ok, nullability is not strongly required.
I can unwrap unneeded numbers to `NaN` or `Zero`. Which better?

Reply:
Unwrapped to `int`. Not provided values threat as `0`, which means `unspecified`.